apps/SimpleExt/SimpleExt.rc

```diff
@@ -93,20 +90,20 @@ VS_VERSION_INFO VERSIONINFO
 BEGIN
     BLOCK "StringFileInfo"
     BEGIN
-        BLOCK "040c04b0"
+        BLOCK "000904b0"
         BEGIN
-            VALUE "FileDescription", "Rename-It! Shell Extension"
-            VALUE "FileVersion", "2, 0, 0, 1"
+            VALUE "FileDescription", "Rename-It! v4.x Shell Extension for x64"
+            VALUE "FileVersion", "4.1.1.1"
             VALUE "InternalName", "SimpleEx"
-            VALUE "LegalCopyright", "Copyright (C) 2003-2005"
+            VALUE "LegalCopyright", "Under Apache License"
```

Comment:
It's still "Copyright (c) 2003-2016 Werner Beroux"

*Adding* that it's under Apache License in the UI like a good follow up improvement, with a link to GitHub.